source/default/_patterns/02-molecules/utchero/_utchero.twig

```diff
@@ -0,0 +1,100 @@
+{#
+/**
+ * @file
+ *   card
+ *
+ * Available variables for card:
+ *  - card.variant [string]: When set, adds variant styling and behavior.
+ *  - card.image [array]: Implements @atoms/image/_image.twig.
+ *  - card.title [string]: The title of the card.
+ *  - card.body [string]: The body content of the card.
+ *  - card.list [array]: A list of terms describing card content.
+ */
+#}
+
+{# Sets colors for hero content depending on selection #}
+	{% if utchero.hero_color == 'Dark Blue' %}
+		{% set title_color = 'lg:text-white' %}
+		{% set body_color = 'lg:text-white' %}
+		{% set btn_border_color = "lg:border-white" %}
+		{% set bg_color = 'background-color:#112e51' %}
+		{% set bg1_opacity = '' %}
+    	{% set bg2_opacity = 'opacity-90' %}
+	{% else %}
+		{% set title_color = 'text-utc-new-blue-500' %}
+		{% set body_color = 'text-base' %}
+		{% set btn_border_color = "lg:border-utc-new-blue-500" %}
+		{% set bg_color = 'background-color:#104dd4' %}
+		{% set bg1_opacity = 'opacity-20' %}
+    {% set bg2_opacity = 'opacity-20' %}
+	{% endif %}
+
+<div class="{{ card_classes }}">
+	{% block utchero_content %}
+  {# image on left #}	 
+	{% if utchero.hero_align == 'Left' %}
+		<div class="lg:grid lg:grid-rows-utchero lg:grid-cols-utchero lg:gap-y-4 lg:grid-flow-row lg:mb-4 text-center">
+    {# TOdo ADD conditional statement if the image contains patternlab data or not. #}
+			{% if utc_hero.hero_image %}
+				<div class="lg:col-start-2 lg:col-end-3 lg:row-start-2 lg:row-end-6 lg:m-auto col-span-1 z-30">
+					{# {{ utchero.hero_image }} #}
+        {% include '@atoms/image/_image.twig' with {
+              image: utchero.hero_image,
+              image_classes: ['w-full']
+        } %}
+          {# <img src='https://via.placeholder.com/1260x840.jpeg'> #}
+				</div>
+			{% endif %}
+				<div class="lg:col-start-3 lg:col-end-4 lg:row-start-1 lg:row-end-4 lg:my-auto lg:ml-8 p-4 z-30">
+	{% else %}
+	{# image on right #}	
+		<div class="lg:grid lg:grid-rows-utchero lg:grid-cols-utcheroright lg:gap-y-6 grid-flow-row lg:mb-4 text-center">
+			{# {% if utc_hero.hero_image %} #}
+				<div class="lg:col-start-3 lg:col-end-4 lg:row-start-2 lg:row-end-5 lg:m-auto z-30">
+					{# {{ utchero.hero_image }} #}
+          {# <img src='https://via.placeholder.com/1260x840.jpeg'> #}
+				</div>
+			{# {% endif %} #}
+				<div class="lg:col-start-2 lg:col-end-2 lg:row-start-1 lg:row-end-4 lg:mr-8 lg:my-auto py-4 px-6 col-span-1 z-30">
+	{% endif %}
+				{% block herocontent %}
+					{% block herotagtitle %}
+						{% if utchero.hero_tag %}
+							<p class="text-left font-semibold tracking-wide uppercase text-lg {{ body_color }}">
+								{{ utchero.hero_tag }}
+							</p>
+							<hr class="border-t-3 border-utc-new-gold-500 mt-2 mb-6 w-20"/>
+						{% endif %}
+						{% if utchero.hero_title %}
+							<h2 class="text-left {{ title_color }}">
```

Comment:
It was a different font in your samples, is that just because the h2 is set different in Patternlab, or are we moving away from Exo? Not that I'm low key loving the font in your screenshots > Exo or anything...;)

Reply:
I'm not sure why the font is different. There are not plans to move yet. It's probably a different config that I got to update.